### PR TITLE
jquery.tablesorter: Fix SPDX license. See #5544

### PR DESCRIPTION
**Pull request by Mottie**

### CODE REVIEW

File: ajax/libs/jquery.tablesorter/package.json [hunk 1]
```diff
@@ -17,10 +17,7 @@
     "alphanumeric",
     "natural"
   ],
-  "licenses": [
-    "MIT",
-    "deprecated_GPL-2.0+"
-  ],
+  "licenses": "(MIT OR GPL-2.0+)",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.tablesorter/package.json]: sorry but we don't support this format but use the array

Comment (Mottie) [inline ajax/libs/jquery.tablesorter/package.json]: Oh, I thought you wanted a SPDX string... I should have used "license" (no "s").

**Comment by PeterDaveHello** [comment]
Though http://git.spdx.org/ is removed ... the last time we synced from upstream (SPDX v2.5) it's:
https://github.com/cdnjs/cdnjs/blob/12d79a4bf555f80bd5b8c1b694ccf127e69aea0c/tools/license-list.json#L113
`  "deprecated_GPL-2.0+",`

File: ajax/libs/jquery.tablesorter/package.json (hunk 1 above)
Comment (Mottie) [inline ajax/libs/jquery.tablesorter/package.json]: If it doesn't matter, then we can just close this PR.

Comment (PeterDaveHello) [inline ajax/libs/jquery.tablesorter/package.json]: but that's not how we use it currently, hmmm ...

Comment (PeterDaveHello) [inline ajax/libs/jquery.tablesorter/package.json]: I think it doesn't really matter, if there is any inconvenience caused, please feel free to discuss with us, thanks!

**Pull request closed without merging**